Shared/Store/NetworkStore.swift

```diff
@@ -41,5 +47,12 @@ open class NetworkStore {
         if !isRunningTest {
             ((try? self.reachability?.startNotifier()) as ()??)
         }
+
+        dispatcher.register
+                .filterByType(class: NetworkAction.self)
+                .filter { $0 == NetworkAction.retry }
+                .map { _ in (self.reachability?.connection ?? Reachability.Connection.none) != .none }
+                .bind(to: self._connectedToNetwork)
+                .disposed(by: self.disposeBag)
```

Comment:
I think this is good, but doesn't overcome the limitations of the reachability API.

Reply:
in practice, the `retry` action is rarely required; the callbacks preceding this subscription actually get fired very quickly!